Tasks/DownloadBuildArtifacts/main.ts

```diff
@@ -231,6 +231,16 @@ async function main(): Promise<void> {
                         reject(reason);
                     }));
                 }
+                else if (artifact.resource.type.toLowerCase() === "drop") {
+                    let properties = {
+                        artifacttype: artifact.resource.type,
+                        artifactname: artifact.name,
+                        itempattern: itemPattern,
+                        manifestid: artifact.resource.data
+                    };
+
+                    tl.command("artifact.download", properties, downloadPath);
```

Comment:
Note, we should not add this command permanently. When this command is used, I would have the command handler check the currently running task and verify the ID is your task otherwise fail. Customers should not be able to use this in their ad hoc scripts. Fire and forget command doesn't make sense for download, from user scripts.

Reply:
should be just switch the entire drop download into C# when GA?